ui/admin/app/components/agent/Publish.tsx

```diff
@@ -22,11 +22,7 @@ type PublishProps = {
     onPublish: (alias: string) => void;
 };
 
-export function Publish({
```

Comment:
The linter was complaining about this.

Reply:
I actually already made this fix, feel free to remove it if it gives you conflicts